babel.dependencies.config.js

```diff
@@ -1,31 +0,0 @@
-/** @format */
-const config = {
-	// see https://github.com/webpack/webpack/issues/4039#issuecomment-419284940
-	sourceType: 'unambiguous',
-	presets: [
-		[
-			'@babel/env',
-			{
-				modules: false,
```

Comment:
This line isn't present in `@automattic/calypso-build/babel/default`: https://github.com/Automattic/wp-calypso/blob/948f23d6a0c7d9243377cb5346997238a160c183/packages/calypso-build/babel/default.js#L4-L10

The default here is [`auto`](https://babeljs.io/docs/en/babel-preset-env#modules), which, when used with `babel-loader` (as is the case here), translates to `false`. Should be good :heavy_check_mark: